recipes/cminpack/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+mkdir build
+cd build
+
+CMAKE_BUILD_TYPE=RelWithDebInfo
+
+cmake -G "Ninja" \
+  -D BUILD_SHARED_LIBS:BOOL=ON \
+  -D CMAKE_PREFIX_PATH=$PREFIX \
+  -D CMAKE_INSTALL_PREFIX:PATH=$PREFIX \
+  -D CMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE \
+  -D CMINPACK_LIB_INSTALL_DIR="lib" \
+  -D USE_BLAS:BOOL=OFF \
```

Comment:
please leave blas enabled, at least on unix, and depend on openblas

Reply:
@xantares, I'd like to not deal with BLAS for now (see original PR message), but please feel free if you'd like to try that ;). My plan was to first release a version without this dependency, and then release a new build number (or new version) in the feedstock, adding BLAS.

May I add you as maintainer, also?

Reply:
why ? its only a few few lines more: just add openblas to host and run dependency with the [unix] selector and switch USE_CBLAS on.

Reply:
because then it would have to always depend on `blas=openblas`, conflicting with whoever uses `blas=mkl` (from Anaconda)

Reply:
no, conda-forge packages are not supposed to be compatible with defaults

Reply:
But a lot of people use a mix of conda-forge + Anaconda, specially because of the numpy/scipy stack built with MKL (e.g., in Windows there even isn't a SciPy in conda-forge). I'm not saying to not add support for `openblas`, but if we could add this support later, it would be better, simplifying the creating of the feedstock and allowing this package to already be used in the situation I described.

Reply:
I'm trying to add OpenBLAS support on Unix, waiting for CI now, but note that a test failed locally here (`tlmdifc`), with some minor differences:

```
--- examples/ref/tlmdifc.ref	2017-02-24 12:09:31.000000000 -0300
+++ build/examples/tlmdifc.out	2019-02-15 10:34:41.363060952 -0200
@@ -9,6 +9,6 @@
           0.08241058       1.133037       2.343695
       covariance
 
-        0.0001531202    0.002869941   -0.002656662
-         0.002869941     0.09480935    -0.09098995
-        -0.002656662    -0.09098995     0.08778728
+        0.0001531203    0.002869942   -0.002656662
+         0.002869942     0.09480937    -0.09098996
+        -0.002656662    -0.09098996     0.08778729
```

I'll try to remove this test when building with OpenBLAS.

Reply:
Done, it's working, building for both `openblas` and `noblas` when in Linux or macOS, and just `noblas` when in Windows